bark/models/behavior/not_started/behavior_not_started.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#ifndef BARK_MODELS_BEHAVIOR_NOT_STARTED_BEHAVIOR_NOT_STARTED_HPP_
+#define BARK_MODELS_BEHAVIOR_NOT_STARTED_BEHAVIOR_NOT_STARTED_HPP_
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "bark/models/behavior/behavior_model.hpp"
+#include "bark/models/dynamic/dynamic_model.hpp"
+
+namespace bark {
+namespace models {
+namespace behavior {
+
+using dynamic::Trajectory;
+using world::ObservedWorld;
+
+// model that always has behavior status not started (to be used in search,
+// replacing BehaviorStaticTrajectory, to prevent that BehaviorStaticTrajectory
+// becomes true in forward search and thus suddenly unexpectingly shows up)
+class BehaviorNotStarted : public BehaviorModel {
```

Comment:
because it is not just data-replay, but closed loop simulation. and I am populating the scene with agents (some of them are ready at t=0, some of them later), but I do not want the agents to become valid during forward search. So for all that are not valid at t=t_planningstart, I replace them by BehaviorNotStarted for that planning loop